### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by uvidit**

### CODE REVIEW

File: src/test/java/weatherApiTests/AuthKeyTest.java [hunk 1]
```diff
@@ -1,46 +1,70 @@
...
-import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@Epic("API tests for testWeather project")
+@Feature("Authentication verification")
 
 public class AuthKeyTest {
-    private EndPoints ed = new EndPoints();
+    private EndPoints endPoint1 = new EndPoints();
```
Comment (uvidit) [inline src/test/java/weatherApiTests/AuthKeyTest.java:18]: Why #1 ? :)

File: src/test/java/weatherApiTests/AuthKeyTest.java [hunk 2]
```diff
@@ -1,46 +1,70 @@
...
+                .statusCode(expectedStatusCodeUnath);
+
+        LOG.debug("Test: It's impossible to get data without APPID");
+    }
+
+    @DataProvider
+    public Object[][] requestData() {
+        return new Object[][]{
+                {null, expectedStatusCodeUnath, "verifyNullAppId"},
+                {"4t5gdg534df453", expectedStatusCodeUnath, "verifyIncorrectAppIdShorl"},
```
Comment (uvidit) [inline src/test/java/weatherApiTests/AuthKeyTest.java]: > verifyIncorrectAppIdShorl
a tyny typo: verifyIncorrectAppIdShor**t**

File: src/test/java/weatherApiTests/AuthKeyTest.java [hunk 3]
```diff
@@ -1,46 +1,70 @@
...
 
-    @DataProvider
-    public Object[][] requestData() {
-        return new Object[][]{
-                {null, 401},
-                {"4t5gdg534df453", 401},
-                {"0b4a9150dfd5e34fc0d8afa2a8d1629f", 401},
-                {"0b4a9150ecd5efe7c0d8afa2a8d1629e", 200}
+    private static final Logger LOG = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger("com.testWeather.AuthKeyTest");
+    private static int expectedStatusCodeUnath = 401;
```
Comment (uvidit) [inline src/test/java/weatherApiTests/AuthKeyTest.java]: > expectedStatusCodeUn**ath**
a tiny typo expectedStatusCodeUna**u**th

File: src/test/java/weatherApiTests/AuthKeyTest.java (hunk 2 above)
Comment (avoloshchuk567) [reply src/test/java/weatherApiTests/AuthKeyTest.java]: Corrected in b7e25e3cc8e28291f710c386ea987844f18e973b

File: src/test/java/weatherApiTests/AuthKeyTest.java (hunk 3 above)
Comment (avoloshchuk567) [reply src/test/java/weatherApiTests/AuthKeyTest.java]: Corrected in b7e25e3cc8e28291f710c386ea987844f18e973b

File: src/test/java/weatherApiTests/AuthKeyTest.java (hunk 1 above)
Comment (avoloshchuk567) [reply src/test/java/weatherApiTests/AuthKeyTest.java:18]: This was a type. 
Has already been corrected.

**Pull request merged**